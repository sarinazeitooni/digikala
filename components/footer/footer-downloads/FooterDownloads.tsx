import style from './style/footer-download.module.scss';
import { v4 as uuidv4 } from 'uuid';
import messages from "./messages/messages";

const FooterDownloads = () => {
    return (
        <div className={style['footer-download-container']}>
            <div className={style['footer-download']}>
                <div className={style['download-title-container']}>
                    <div className={style['label']}>{messages.downloadTitle}</div>
                    <img src='https://www.digikala.com/static/files/59d36e01.svg'/>
                </div>
                <div className={style['downloads']}>
                    <div className={style['download-more']}> ... </div>
                    {messages.urls.map((url)=>{
                        return(
                            <img key={uuidv4()} src={url} className={style['download-item']}/>
                        )
                    })}
                </div>
            </div>
        </div>
    )
}
export default FooterDownloads;