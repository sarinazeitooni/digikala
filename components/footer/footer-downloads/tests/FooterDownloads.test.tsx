import {shallow} from "enzyme";
import FooterDownloads from "../FooterDownloads";
describe("FooterDownloads component" , ()=>{
    it("should render correctly" , ()=>{
        const container = shallow(<FooterDownloads/>);
        expect(container).toMatchSnapshot();
    })
});