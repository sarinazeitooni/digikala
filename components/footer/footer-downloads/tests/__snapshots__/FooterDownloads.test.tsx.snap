// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterDownloads component should render correctly 1`] = `ShallowWrapper {}`;
